create-wundergraph-app fails with `Error: Cannot find module 'dotenv'`

### Bug description

When trying to follow the getting started instructions on https://docs.wundergraph.com/getting-started, `create-wundergraph-app` is unsuccessful.

### How to reproduce

1. run `npx create-wundergraph-app my-project -E simple`
2. observe error: `Error: Cannot find module 'dotenv'`

### Expected behavior

A minimal wundergraph project should be created

### WunderGraph information

N/A

### Environment & setup

- OS: Mac OS Monterey 12.6.1
- Go version: go version go1.19.4 darwin/arm64
- Database: N/A
- Node.js version: v19.0.0


### WunderCtl Version

N/A

--- a/packages/create-wundergraph-app/package.json
+++ b/packages/create-wundergraph-app/package.json
@@ -30,6 +30,7 @@
     "async-retry": "^1.3.3",
     "chalk": "^4.1.2",
     "commander": "^9.4.1",
+    "dotenv": "^16.0.3",
     "figlet": "^1.5.2",
     "got": "^11.8.5",
     "gradient-string": "^2.0.2",
@@ -46,7 +47,6 @@
     "@types/jest": "^29.2.2",
     "@types/node": "^18.11.8",
     "@types/tar": "^6.1.3",
-    "dotenv": "^16.0.3",
     "jest": "^29.3.1",
     "ts-jest": "^29.0.3",
     "typescript": "^4.8.4"

